doc/contributing/distribution.md

```diff
@@ -0,0 +1,24 @@
+# Node.js Distribution Policy
+
+This document describes some policies around what is and is not included in the
+Node.js distribution.
+
+## Inclusion
+
+Node.js includes some external projects that the Node.js team does not maintain.
+The fact of a project's inclusion should not imply anything about the project
+relative to its competitors; in some cases, a project was added when it had no
+competitors. While the Node.js team supports and encourages competition in the
+JavaScript ecosystem, as a policy, the Node.js project does not include multiple
+dependencies or tools that serve the same purpose.
+
+The following user-accessible external projects are the ones chosen for their
+particular purposes:
+
+* JavaScript engine: V8
+* Package manager: `npm`
```

Comment:
```suggestion
* Package manager: `npm`
* Package manager manager: Corepack
* `fetch` implementation: Undici
```

Reply:
I think Corepack is more properly described as a package manager version manager.

Undici isn’t exposed to users, is it? I was only listing the visible ones.

Reply:
I'd argue it is, via some "global symbols"

Reply:
```suggestion
* Package manager: `npm`
* Package manager version manager: Corepack
```